sources/Algo/include/GeneratorDefinitionAlgorithm.h

```diff
@@ -36,15 +36,18 @@ class GeneratorDefinition {
    * @brief Generator model type
    */
   enum class ModelType {
-    SIGNALN_INFINITE = 0,        ///< Use GeneratorPVSignalN model with infinite diagram
-    SIGNALN_RECTANGULAR,         ///< Use GeneratorPVSignalN model with rectangular diagram
-    DIAGRAM_PQ_SIGNALN,          ///< Use GeneratorPVDiagramPQSignalN model
-    REMOTE_SIGNALN_INFINITE,     ///< Use GeneratorPVRemoteSignalN
-    REMOTE_SIGNALN_RECTANGULAR,  ///< Use GeneratorPVRemoteSignalN with rectangular diagram
-    REMOTE_DIAGRAM_PQ_SIGNALN,   ///< Use GeneratorPVRemoteDiagramPQSignalN
-    PROP_SIGNALN_INFINITE,       ///< Use GeneratorPQPropSignalN
-    PROP_SIGNALN_RECTANGULAR,    ///< Use GeneratorPQPropSignalN with rectangular diagram
-    PROP_DIAGRAM_PQ_SIGNALN,     ///< Use GeneratorPQPropDiagramPQSignalN
+    SIGNALN_INFINITE = 0,        ///< Use signalN generator model with infinite diagram
+    SIGNALN_RECTANGULAR,         ///< Use signalN generator model with rectangular diagram
+    DIAGRAM_PQ_SIGNALN,          ///< Use signalN generator model
+    SIGNALN_TFO_INFINITE,        ///< Use signalN generator model with infinite diagram and tfo
+    SIGNALN_TFO_RECTANGULAR,     ///< Use signalN generator model with rectangular diagram and tfo
+    DIAGRAM_PQ_TFO_SIGNALN,      ///< Use signalN generator model and tfo
```

Comment:
just to have comments very explicit, could you replace tfo by transformers in comments? thx :)